liquibase-core/src/main/java/liquibase/structure/core/Table.java

```diff
@@ -31,6 +31,15 @@ public List<ForeignKey> getOutgoingForeignKeys() {
         return getAttribute("outgoingForeignKeys", List.class);
     }
 
+    public String getTableSpace() {
+        return getAttribute("tableSpace", String.class);
+    }
+
+    public Relation setTableSpace(String remarks) {
```

Comment:
We need to rename this method to `setTablespace`
To make it consistent with other classes

Reply:
Done, thanks!